content/blog/projects/current/F1-10.md

```diff
@@ -0,0 +1,62 @@
+---
+lastmod: 2023-09-07T12:23:00Z
+head:
+  meta:
+    - name: keywords
+      content: project, F1TENTH, in-progress, robots, bolid
+    - name: robots
+      content: index follow
+    - name: author
+      content: Aleksandra Lis
+    - name: copyright
+      content: © 2023 KN Integra
+    - name: createdAt
+      content: 2023-09-07T12:23:00Z
+---
+
+# F1TENTH
+
+F1TENTH AGH to autonomiczny bolid elektryczny zbudowany na wzór bolidów wyścigowych, jednak w skali 1:10. Kod sterujący oraz zarządzający wszystkimi elementami został napisany w języku Python i C++. Oprogramowanie tworzone jest w oparciu o środowisko ROS (Robot Operating System), które zarządza odbieraniem i wysyłaniem informacji do poszczególnych urządzeń peryferyjnych.  
+
+Bolid wyposażony jest w dwa niezależne źródła zbierania danych o otoczeniu – LIDAR 2D Hokuyo UST-10LX i kamerę głębi Intel RealSense D435i. Sercem pojazdu jest komputer pokładowy Nvidia Jetson TX2, na którym wykonywane są wszystkie obliczenia pozwalające na autonomiczne poruszanie się.
+
+Aktualnie opracowany algorytm sterowania to tzw. algorytm reaktywny. Dzięki zastosowaniu takiej koncepcji bolid jest w stanie poruszać się po nieznanym wcześniej, zamkniętym torze (nie ma potrzeby wcześniejszego skanowania otoczenia). Więcej na temat algorytmu można przeczytać w artykule: https://doi.org/10.1007/978-3-031-35173-0_22
```

Comment:
Więcej na temat algorytmu można przeczytać w artykule: [Reactive Control Algorithm for F1TENTH Autonomous Vehicles in Unknown Dynamic Environments](https://doi.org/10.1007/978-3-031-35173-0_22)